networking/network_security/AdminNetworkPolicy/ovn-k-egress-nodes-networks-peer.adoc

```diff
@@ -0,0 +1,9 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ovn-k-egress-nodes-networks-peer"]
+= Egress nodes and networks peer for AdminNetworkPolicy and BaselineAdminNetworkPolicy
+include::_attributes/common-attributes.adoc[]
+:context: ovn-k-egress-nodes-networks-peer
+
+toc::[]
+
```

Comment:
IMO you should add a short abstract/introduction here that will precede the module, so that there's something between the main page heading and the module subheading.